### PR TITLE
Fix h2o-mojo-compatibility error

### PR DESCRIPTION
**Pull request by wendycwong**
Changed Mojo version of StackedEnsemble from 1.0 to 1.00.

### CODE REVIEW

File: h2o-algos/src/main/java/hex/ensemble/StackedEnsembleMojoWriter.java [hunk 1]
```diff
@@ -3,11 +3,11 @@
 import hex.Model;
 import hex.MultiModelMojoWriter;
 import hex.StackedEnsembleModel;
+import water.DKV;
```
Comment (navdeep-G) [inline h2o-algos/src/main/java/hex/ensemble/StackedEnsembleMojoWriter.java]: Why this import?

Comment (wendycwong) [reply h2o-algos/src/main/java/hex/ensemble/StackedEnsembleMojoWriter.java]: Not sure, I did not touch that part.

Comment (navdeep-G) [reply h2o-algos/src/main/java/hex/ensemble/StackedEnsembleMojoWriter.java]: Or rather, why was it added if it is already there?

Comment (wendycwong) [reply h2o-algos/src/main/java/hex/ensemble/StackedEnsembleMojoWriter.java]: Wait, it is there in master originally.  IntelliJ changed its location.

Comment (navdeep-G) [reply h2o-algos/src/main/java/hex/ensemble/StackedEnsembleMojoWriter.java]: Yep, agree @michalkurka

**Pull request merged**